admin/config.yml

```diff
@@ -12,6 +12,13 @@ public_folder: /assets/blog
 publish_mode: editorial_workflow
 logo_url: https://18f.gsa.gov/assets/img/logos/18f-logo.svg
 
+autoprefixer:
```

Comment:
I think this config needs to be in the `_config.yml`. This config is for the Netlify CMS and not Jekyll.

Reply:
Oh my mistake, thanks for catching that.

Reply:
Awesome!